web/src/components/tabs/Tab/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { createUseStyles } from 'react-jss'
+
+import tokens from '../../../tokens'
+
+export default createUseStyles((theme: typeof tokens) => ({
+  tab: ({ active, align }) => ({
+    background: 'none',
+    zIndex: 1,
```

Comment:
do we need that zIndex? it should be avoided if possible

Reply:
> * Tab selection shouldn't work via `Tab` key, but via arrow keys (right, left). The `Tab` key is preserved for focusing contents within and below the active tab panel, and `Shift`+`Tab` returns the user to the active tab. (-> implementation help: https://inclusive-components.design/tabbed-interfaces/).
> * Focus style (focus ring) should only be visible when navigating with the keyboard, not the mouse (-> `focus-visible`).
> * The active tab should have `aria-selected` set to `true`.
> * The indicator line should have the same width as the text above and not extend it (see design).
> * Spacing between tab text and indicator/divider line is too small (see design).
> * Tabs in small size show no `cursor: pointer` when hovering the text (only when hovering the small indicator line), but should and behave the same as the other sizes.
> * Do we need the `margin` prop? We should use spacing components instead for consitent spacing and keeping our components encapsulated, so I'd remove it.

Fixed almost everything from this list, I have some questions

* Can you please design the focus ring? Because there's no default one for `focus-visible`, and I just did something similar to a normal focus ring, but I guess anyway it looks unsuitable.
* No we don't need `margin` prop here, same as `width`. But I added them just to make the component looks pretty in the story. Now I extended the story root block to full size and for me, it looks a bit weird. If it looks ok for you in the story, then we're good to merge it I guess. Otherwise, I can return `width` and `margin` and make it prettier (from my view), as it was before.

Reply:
Unfortunately, I don't see any other solution here, how I can overlay a dark grey indicator line for the tab on hover

Reply:
See other comment of `:after` element